src/components/articles/createArticle.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import NewArticle from "../../views/articles/createArticle";
+import { connect } from "react-redux";
+import {
+  createNewArticleAction,
+  publishNewArticleAction
+} from "../../actions/articleActions/articleAction";
+import PropTypes from "prop-types";
+import { myHeaders } from "../../utils/myHeaders";
+import { toast } from "react-toastify";
+
+class CreateArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allcategory: [],
+      title: "",
+      description: "",
+      body: "",
+      category: "",
+      tags: []
+    };
+    this.handleEditorChange = this.handleEditorChange.bind(this);
+    this.componentDidMount = this.componentDidMount.bind(this);
+    this.handleSelectChange = this.handleSelectChange.bind(this);
+    this.handlePublish = this.handlePublish.bind(this);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
There's no need for binding 'this' when you're using arrow functions. Please remove that.

Reply:
The binding has been removed. The arrow functions don't need the bind